README.md

```diff
@@ -1431,29 +1477,35 @@ features: {
+- **features.webauthn** - Display and use factors supported by the FIDO 2.0 (Web Authentication) security standard. Enabling this feature will prevent the widget from invoking the legacy Windows Hello factor. Defaults to `false`. **Note:** Not supported in v2.
 
-- **features.selfServiceUnlock** - Display the "Unlock Account" link to allow users to unlock their accounts. Defaults to `false`.
+- **features.selfServiceUnlock** - Display the "Unlock Account" link to allow users to unlock their accounts. Defaults to `false`. **Note:** Not supported in v2. In v2, requires Widget to be Okta-hosted for additional configuration.
 
-- **features.multiOptionalFactorEnroll** - Allow users to enroll in multiple optional factors before finishing the authentication flow. Default behavior is to force enrollment of all required factors and skip optional factors. Defaults to `false`.
+- **features.multiOptionalFactorEnroll** - Allow users to enroll in multiple optional factors before finishing the authentication flow. Default behavior is to force enrollment of all required factors and skip optional factors. Defaults to `false`. **Note:** Not supported in v2. In v2, requires Widget to be Okta-hosted for this configuration.
 
 - **features.hideSignOutLinkInMFA** - Hides the sign out link for MFA challenge. Defaults to `false`.
 
-- **features.registration** - Display the registration section in the primary auth page. Defaults to `false`.
+- **features.registration** - Display the registration section in the primary auth page. Defaults to `false`. **Note:** Not supported in v2. In v2, requires Widget to be Okta-hosted for this configuration.
+
+- **features.idpDiscovery** - Enable [IdP Discovery](#idp-discovery). Defaults to `false`. **Note:** Supported by default in v2 and feature flag is not required.
 
-- **features.idpDiscovery** - Enable [IdP Discovery](#idp-discovery). Defaults to `false`.
+- **features.showPasswordToggleOnSignInPage** - End users can now toggle visibility of their password on the Okta Sign-In page, allowing end users to check their password before they click Sign In. This helps prevent account lock outs caused by end users exceeding your org's permitted number of failed sign-in attempts. Note that passwords are visible for 30 seconds and then hidden automatically. Defaults to `false`. **Note:** Defaults to `true` in v2.
 
-- **features.showPasswordToggleOnSignInPage** - End users can now toggle visibility of their password on the Okta Sign-In page, allowing end users to check their password before they click Sign In. This helps prevent account lock outs caused by end users exceeding your org's permitted number of failed sign-in attempts. Note that passwords are visible for 30 seconds and then hidden automatically. Defaults to `false`.
+## Widget feature support table
+| feature     | v1 | v2  |
+| ----------- | ---|-----|
+| router      | ✅  | ❌  |
+| rememberMe   | ✅ | ✅    |
```

Comment:
Not finished because not sure if this helpful or not. It would be nice to have a table to do quick lookups without having to read a paragraph.